scripts/collect-svg-icons.js

```diff
@@ -0,0 +1,124 @@
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+const path = require( 'path' );
+const sharp = require( 'sharp' );
+
+const ROOT_DIRECTORY = path.join( __dirname, '..' );
+const BUILD_DIRECTORY = path.join( ROOT_DIRECTORY, 'build' );
+const ICONS_DIRECTORY = path.join( BUILD_DIRECTORY, 'icons' );
+
+collectSvgIcons();
+
+async function collectSvgIcons() {
+	if ( !fs.existsSync( BUILD_DIRECTORY ) ) {
+		fs.mkdirSync( BUILD_DIRECTORY );
+	}
+
+	if ( fs.existsSync( ICONS_DIRECTORY ) ) {
+		fs.rmSync( ICONS_DIRECTORY, { recursive: true, force: true } );
+	}
+
+	fs.mkdirSync( ICONS_DIRECTORY );
+
+	const svgFilesData = glob.sync( '**/packages/*/theme/icons/*.svg', {
+		cwd: ROOT_DIRECTORY,
+		ignore: [ '**/build/**', '**/node_modules/**' ]
+	} ).map( svgFilePath => {
+		const normalizedPath = normalizePath( svgFilePath );
+		const [ , packageName, originalFilename ] = normalizedPath.match( /packages\/([^/]+)\S+\/([^/]+)\.svg/ );
+		const originalPath = normalizePath( path.resolve( svgFilePath ) );
+		const newFilename = `${ packageName }___${ originalFilename }.png`;
+		const newPath = path.join( ICONS_DIRECTORY, newFilename );
+		const shorthand = `{@icon @ckeditor/${ normalizedPath.split( 'packages/' )[ 1 ] }}`;
+		const isExternal = normalizedPath.includes( 'external/' );
+
+		return { packageName, originalFilename, originalPath, newFilename, newPath, shorthand, isExternal };
+	} ).sort( ( a, b ) => {
+		return a.isExternal - b.isExternal ||
+		a.packageName.localeCompare( b.packageName ) ||
+		a.originalFilename.localeCompare( b.originalFilename );
+	} );
+
+	for ( const svgFileData of svgFilesData ) {
+		await sharp( svgFileData.originalPath ).png().toFile( svgFileData.newPath );
+	}
+
+	const htmlPath = normalizePath( path.join( ICONS_DIRECTORY, 'index.html' ) );
+	fs.writeFileSync( htmlPath, createHtmlContent( svgFilesData ), 'utf-8' );
+
+	console.log( chalk.green( `Summary: ${ chalk.underline( htmlPath ) }` ) );
+}
+
+function normalizePath( _path ) {
+	return _path.split( path.sep ).join( path.posix.sep );
+}
+
+function createHtmlContent( svgFilesData ) {
+	const svgFileDataByPackage = svgFilesData.reduce( ( svgFileDataByPackage, svgFileData ) => {
+		if ( !svgFileDataByPackage[ svgFileData.packageName ] ) {
+			svgFileDataByPackage[ svgFileData.packageName ] = [];
+		}
+
+		svgFileDataByPackage[ svgFileData.packageName ].push( svgFileData );
+
+		return svgFileDataByPackage;
+	}, {} );
+
+	return [
+		'<!DOCTYPE html>',
+		'<html>',
+		'	<head>',
+		'		<style>',
+		'			body { font-family: arial }',
+		'			table { border-collapse: collapse; margin: auto; margin-bottom: 25vh; }',
+		'			thead tr:first-child th { padding-top: 20px; font-size: 150%; text-transform: uppercase; }',
+		'			td { border: 1px solid black; }',
+		'			img { max-height: 20px; }',
```

Comment:
```suggestion
		'			* { box-sizing: border-box; }',
		'			body { font-family: Arial; }',
		'			table { border-collapse: collapse; margin: auto; margin-bottom: 25vh; }',
		'			thead tr:first-child th { padding-top: 20px; font-size: 150% }',
		'			tbody tr:nth-child(odd) td {background: #e5e5e5;}',
		'			td { border: 1px solid black; padding: 2px; }',
		'			img { max-height: 20px; }',
```

<img width="867" alt="image" src="https://user-images.githubusercontent.com/2270764/216448260-f5f83210-6514-4dc6-8753-53c69cbfe62b.png">